Wake On Lan/Modules/AboutScreen/Package.swift

```diff
@@ -0,0 +1,38 @@
+// swift-tools-version:5.3
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "AboutScreen",
+    platforms: [.iOS(.v13)],
+    products: [
+        // Products define the executables and libraries a package produces, and make them visible to other packages.
+        .library(
+            name: "AboutScreen",
+            targets: ["AboutScreen"])
+    ],
+    dependencies: [
+        // Dependencies declare other packages that this package depends on.
+        // .package(url: /* package url */, from: "1.0.0"),
+        .package(path: "../WOLResources"),
+        .package(path: "../WOLUIComponents"),
+        .package(path: "../SharedProtocols"),
+        .package(path: "../SharedExtensions"),
```

Comment:
Здесь потеряли `RouterProtocol`
```suggestion
        .package(path: "../SharedExtensions"),
        .package(path: "../RouterProtocol"),
```